project/src/main/puzzle/piece/frame-input.gd

```diff
@@ -13,36 +13,48 @@ export (String) var action: String
 # 'ui_left' no longer gets triggered.
 export (String) var cancel_action: String
 
+var pressed_frames: int
+
 var _just_pressed: bool
 var _pressed: bool
-
-var frames: int
 var _buffer: bool
+var _print_inputs: bool = false
```

Comment:
Remove redundant `= false`

Reply:
Changed to `var _print_inputs := false`; this makes it slightly more intuitive to toggle it between true and false, while still avoiding the redundant `bool = false`